packages/components/src/components/Forms/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -0,0 +1,102 @@
+import { View, ViewStyle } from 'react-native'
+import { spacing } from '@department-of-veterans-affairs/mobile-tokens'
+import React, { FC, Fragment } from 'react'
+
+import { Checkbox } from '../Checkbox/Checkbox'
+import { ComponentWrapper } from '../../../wrapper'
+import { Error, Header, Hint } from '../FormText'
+import { FormElementProps } from '../../../types/forms'
+import { Spacer } from '../../Spacer/Spacer'
+import { TextWithA11yAndValue } from '../../../types/common'
+import { useTheme } from '../../../utils'
+
+export type CheckboxGroupProps = FormElementProps & {
+  /** Array of checkbox options. Can be an array containing strings or objects if values or a11y overrides are needed */
+  items: string[] | TextWithA11yAndValue[]
+  /** Callback function that receives an updated array of selected values when checkboxes are pressed */
+  onSelectionChange: (selected: (string | number)[]) => void
+  /** Array of the labels or values (if provided) of currently selected checkboxes */
+  selectedItems: (string | number)[]
+}
+
```

Comment:
Should we have additional documentation here explaining how to configure `onSelectionChange` and `selectedItems` with useState? When used outside FormWrapper at least since I assume that's baked in for it.

Reply:
Added documentation and examples for onSelectionChange, selectedItems, and using TextWithA11yAndValue